src/main/scala/com/redis/Log.scala

```diff
@@ -7,26 +7,26 @@ trait Log {
 
  def ifTrace(message: => String): Unit = if (log.isTraceEnabled) trace(message)
  def trace(message:String, values:AnyRef*): Unit =
-     log.trace(message, values)
+     log.trace(message, values.asInstanceOf[Seq[AnyRef]]: _*)
```

Comment:
I think `.asInstanceOf[Seq[AnyRef]]` is unnecessary

Reply:
Oh, Thank you @xuwei-k san!
 I agree with you. I will remove `.asInstanceOf[Seq[AnyRef]]`.